schemas/dependencies/glitchtip-project-1.yml

```diff
@@ -13,6 +13,8 @@ properties:
     "$ref": "/common-1.json#/definitions/labels"
   name:
     "$ref": "/common-1.json#/definitions/identifierLowercase64"
+  glitchtipId:
```

Comment:
nit: wondering if `projectId` might describe this more accurately.

Reply:
`projectId` is much better! 👍